### PR TITLE
data: AKHIL-GIREESH

### PR DESCRIPTION
**Pull request by AKHIL-GIREESH**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5353"></a>

### CODE REVIEW

**Review by Deva-1903 — DISMISSED** [review]

File: data/AKHIL-GIREESH.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  "bio": "19 | Incipient Web 3 Developer | Blockchain & AI Enthusiast | Amateur Surrealism Artist",
+  "links": [
+    {
+      "name": "Reach out to me on Twitter",
+      "url": "https://twitter.com/_0x416B68696C_",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Reach out to me on Github",
+      "url": "https://twitter.com/_0x416B68696C_",
```
Comment (Sindhuinti) [inline data/AKHIL-GIREESH.json:13]: Please kindly add your github link here (not twitter)

File: data/AKHIL-GIREESH.json [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Reach out to me on LinkedIn",
+      "url": "https://twitter.com/_0x416B68696C_",
```
Comment (Sindhuinti) [inline data/AKHIL-GIREESH.json:18]: Please kindly add your LInkedin link here (not twitter)

**Review by Sindhuinti — CHANGES_REQUESTED** [review]

File: data/AKHIL-GIREESH.json (hunk 2 above)
Comment (AKHIL-GIREESH) [reply data/AKHIL-GIREESH.json]: Fixed! Thank you for pointing it out

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Welcome to LinkFree! 🎉

**Pull request merged**